rust/src/builders/witness_builder.rs

```diff
@@ -388,7 +402,7 @@ impl TransactionWitnessSetBuilder {
         result
     }
 
-    fn remaining_wits(&self) -> RequiredWitnessSet {
+    pub(crate) fn remaining_wits(&self) -> RequiredWitnessSet {
```

Comment:
Again, let's not make functions `pub(crate)` unless we have a reason to

Reply:
updated